### ISSUE TITLE
nxmhandler.exe processes spawn non stop when clicking download with manager

### ISSUE THREAD

**Issue by NSD4rKn3SS** [TanninOne/modorganizer#841]
After switching what game i want to manage from Fallout 4 to Oblivion, I set up where i want my folders and forgot to configure my nexus account, and went to download some mods, when nothing happend i figured i forgot it.

So i did configure that too but my pc suddenly became slow, i checked what's causing it and i had more than a hundred nxmhandler.exe processes and more still spawning until there i ran out of memory and had to do a hard reset. 

Now no matter what game i try to manage this happens.

**Comment by LawLeewer** [comment]
Same thing here.   Only game that I can download mods from nexus through MO is Fallout 4, any other spawns infinite nxmhandler processes.

I use the 1.3 version for the other games. Already tried setting the nxmhandlers from each MO install to handle the links for them individually and also tried setting the 2.0 handler to manage all links.  Get the same result nonetheless.

**Comment by NSD4rKn3SS** [comment]
Since yesterday, i found a workaround, if you changed the game, exit MO, rename or move nxmhandler.exe from MO folder, start ModOrganizer, Go into Settings>Nexus tab> and click Clear Cache, now put or rename nxmhandler.exe back, and it will work again as intended

**Comment by TanninOne** [comment]
Try to start nxmhandler directly, you should get a gui showing which game maps to which mo/nmm install. I assume there is an entry mapping to nxmhandler.exe instead of modorganizer.exe

**Comment by NSD4rKn3SS** [comment]
I'll check it when i get home, though i did edit nxmhandlers ini file yesterday and it only had 4 rows, MO had Fallout 4 associated to it, everything else was set to NMM.
I changed it so every game is associated to MO only, didn't know the program has a gui.

**Comment by NSD4rKn3SS** [comment]
![mo](https://cloud.githubusercontent.com/assets/20150855/16384007/b0428d9c-3c87-11e6-8fda-6640a49bf68e.png)
@TanninOne I'm finally home, and it did had an entry of nxm, so the question is, can we fix this problem somehow ourselves?

**Comment by TanninOne** [comment]
Yes, you can simply double-click on the binary name to change it.Replace "nxmhandler.exe" by "ModOrganizer.exe" and you should be good.

**Comment by NSD4rKn3SS** [comment]
Thank you so much! It's working now.

**Comment by gearhog** [comment]
This almost destroyed my computer... I did what NSD4rKn3SS posted in a previous comment (renaming the nxmhandler.exe, clearing cache) and it now seems to work. I'd tried clearing the cache, and was just about to do this. But someone verifying it working made me feel more confident in not making my PC explode.
Just made this account to thank you! Recently had to format my hard drive and have been running into all kinds of weird issues like this.

**Comment by NSD4rKn3SS** [comment]
I'm glad We could help, and I appreciate your feedback.
